src/DFGNode.cpp

```diff
@@ -527,6 +528,10 @@ void DFGNode::cutEdges() {
   }
 }
 
+void DFGNode::addConst(int t_const){ // combineForUnroll
```

Comment:
Didn't u see there is already an `addConst()`, can you distinguish them by appropriate naming?